Generator/Templates/Shared/delegate.sbntxt

```diff
@@ -3,9 +3,6 @@ using System.Runtime.InteropServices;
 
 namespace {{ namespace }}
 {
-{{
-include 'helper.sbntxt'
-this | write_summary '    '
-~}}
-    public delegate {{ return_value | resolve_type }} {{ name | fix_identifier }}({{ this | write_parameters }});
+    {{ include 'header.sbntxt' this ~}}
+    public delegate {{ return_value | resolve_type }} {{ name | fix_identifier }}({{ this | get_parameters }});
```

Comment:
The delegate get one more indentation level (4 spaces) in the generated code

Reply:
This looks good for me? Do you have an example?

Reply:
@badcel I've discovered that this issue appear only when the delegate doesn't have a doc comment (for example some delegates in Gtk3)